template/template.go

```diff
@@ -192,6 +192,24 @@ func NewTemplateExpander(
 				}
 				return host
 			},
+			"stripDomain": func(hostPort string) string {
+				ret := ""
+				host, port, err := net.SplitHostPort(hostPort)
+				if err != nil {
+					host = hostPort
+				}
+				ip := net.ParseIP(host)
+				if ip != nil {
+					ret = hostPort
+					return ret
+				}
+				ss := strings.Split(host, ".")
+				ret = ss[0]
+				if port != "" {
+					ret = net.JoinHostPort(ss[0], port)
+				}
+				return ret
```

Comment:
```suggestion
				host, port, err := net.SplitHostPort(hostPort)
				if err != nil {
					host = hostPort
				}
				ip := net.ParseIP(host)
				if ip != nil {
					return hostPort
				}
				host = strings.Split(host, ".")[0]
				if host != "" {
					return net.JoinHostPort(host, port)
				}
				return host
```

Reply:
Let's get rid of ret.

Reply:
Ok, I've already added this suggestion to batch